apigentools/utils.py

```diff
@@ -221,17 +232,17 @@ def write_full_spec(config, spec_dir, version, full_spec_file):
             else:
                 full_spec["paths"].update(loaded.get("paths", {}))
                 full_spec["tags"].extend(loaded.get("tags", []))
```

Comment:
We should also check for duplicate tags, even if this is not dict - having tags with same names would still be wrong.